### DISCUSSION TITLE
Rollout of the new version of the INSPIRE Geoportal

### DISCUSSION THREAD

**Discussion by jescriu** [INSPIRE-MIF/helpdesk-geoportal#167] [Announcements]
Dear Community,

The revamped INSPIRE Geoportal based on GeoNetwork is being rolled out in the next days, substituting the Classic Geoportal.
As a result, users of the Geoportal may experience malfunctioning during the next hours, or even temporal unavailability on the site.

Once the new system is in production, the MS / countries users for accessing the GeoNetwork harvesting console awill remain disabled until all the rollout tests have been successfully passed.

We will keep you posted on the progress.
Thank you in advance for your patience.

**Comment by dhdeangelis** [comment]
Looks already very good!  But the new Geoportal seems to lack a "Harvesting status" feature like the old one. Will it include one?

**Comment by MPmurt** [comment]
Is the login logic still the same as I can't login with the old credentials?
